card-picker/card-picker/Card.swift

```diff
@@ -7,8 +7,23 @@
 
 import Foundation
 
+enum CardStates {
+    case faceUp, faceDown, matched
+}
+
 struct Card {
-    var isFaceUp = false
-    var isRemoved = false
-    var emoji: String
+    var state: CardStates = .faceDown
+    var emoji = String()
+    var identifier: Int
+    static var identifierFactory = 0
+    static var lock = true
```

Comment:
If we are not using lock, Please consider removing it.

Reply:
Ok